src/renault_api/kamereon/models.py

```diff
@@ -219,6 +220,15 @@ def warns_on_method(self, method: str) -> Optional[str]:
             )
         return None  # pragma: no cover
 
+    def uses_endpoint_via_kcm(self, endpoint: str) -> bool:
+        """Return True if model uses endpoint via kcm."""
+        # Default to False for unknown vehicles
+        if self.model and self.model.code:
+            return VEHICLE_SPECIFICATIONS.get(self.model.code, {}).get(
+                f"{endpoint}-via-kcm", False
+            )
+        return False  # pragma: no cover
```

Comment:
I think you are adding tests for this, so no need to exclude coverage:
```suggestion
        return False
```